src/main/resources/api/service_openapi.yaml

```diff
@@ -924,7 +924,7 @@ components:
       properties:
         name:
           $ref: "#/components/schemas/Name"
-        referenceDescription:
+        description:
           type: string
```

Comment:
can you document here that the default value is the empty string? (does openapi allow that?)